test/api_test.py

```diff
@@ -19,11 +19,18 @@
 import logging
 import unittest
 
+import googleapiclient.errors
+import mock
+import nose.tools
+
 import hyou.api
 
 import http_mocks
```

Comment:
I see, feel free to leave it as your originally had, seems like the most correct way.